### ISSUE TITLE
Matching pointer file id with dumped file names

### ISSUE THREAD

**Issue by volmacht** [DerPopo/UABE#277]
When a monobehaviour is dumped to text, pointers contain a File ID and a Path ID, but the filename contains the File name and Path ID.

I'm building a tool to import dumped text files into a SQL database, but I can't link the pointer info to the file names. Is there a programmatic way to either convert between file name/ID or to dump with the file ID attached?

**Comment by volmacht** [comment]
My previous issue can be worked around by exporting the map in `View` -> `Dependencies`.

It's still not possible to get the full asset metainfo in the dumped files. It would be really nice to have the info in the window below come with the text dump, either inside the file or in the filename. This would make it possible to consume the dumped assets and map their relationships.

![image](https://user-images.githubusercontent.com/49989705/56844576-83ed9980-6880-11e9-97f5-f5dbc6759f7a.png)

**Comment by DerPopo** [comment]
I agree that having the MonoBehaviour type name in the dumps would be useful. 

Since MonoBehaviour assets have a script reference, you should also be able to extract the class name, namespace and assembly name by searching for the matching MonoScript dump.
